core-drawer-panel.html

```diff
@@ -188,157 +149,212 @@
        * @param {Object} detail.item the panel that the event refers to
        */
 
-      published: {
+      properties: {
+
+        coreSelectorClass: {
+          computed: 'computeCoreSelectorClass(narrow, transition, dragging, rightDrawer)'
+        },
 
         /**
-         * Width of the drawer panel.
+         * The panel to be selected when `core-drawer-panel` changes to narrow
+         * layout.
          *
-         * @attribute drawerWidth
+         * @attribute defaultSelected
          * @type string
-         * @default '256px'
+         * @default 'main'
          */
-        drawerWidth: String,
+        defaultSelected: {
+          type: String,
+          value: 'main'
+        },
 
         /**
-         * Max-width when the panel changes to narrow layout.
+         * If true, swipe from the edge is disable.
          *
-         * @attribute responsiveWidth
-         * @type string
-         * @default '640px'
+         * @attribute disableEdgeSwipe
+         * @type boolean
+         * @default false
          */
-        responsiveWidth: String,
+        disableEdgeSwipe: {
+          type: Boolean,
+          value: false
```

Comment:
I believe they are always falsey as a rule (an absent attribute implies false). I agree about obviousness. However, if that's the convention we should probably also give `disableSwipe` a default value.